README.md

```diff
@@ -125,11 +125,11 @@ export default class App extends Component {
       bccRecipients: ['supportBCC@example.com'],
       body: '<b>A Bold Body</b>',
       isHTML: true,
-      attachment: {
+      attachments: {[
```

Comment:
Need to switch the order of the curly and square brackets.

Reply:
Thanks will sort now..